ai-platform-unified/notebooks/official/featurestore_movie_prediction.ipynb

```diff
@@ -0,0 +1,1342 @@
+{
```

Comment:
<p>(global comment) "Feature Store" (the feature) is written as two words. "Featurestore" the resource is written as one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/283/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Are you suggesting we use Featurestore across the NB?</p>

Reply:
It's context dependent. If we're referring to the feature, it's Feature Store. If we're referring to the resource (the top-level container), it's featurestore.

Reply:
<p>fixed according to the context, ptal</p>